website/docs/content/4. Other/03-image-handling.md

```diff
@@ -0,0 +1,176 @@
+# Image Handling
+
+## Abstract
+
+This document provides an overview of image handling, a crucial component in the creation
+of compute instances. As a fundamental requirement, every compute instance relies on an
+image to boot and function properly. An image refers to a pre-configured and preinstalled
+software package, or an installable operating system, that is compatible with the underlying
+architecture and can be executed by the bootloader. The image serves as the foundation for
+the compute instance, enabling the deployment of various operating systems, applications,
+and services.
+
+## Images are global
```

Comment:
Hi Vincent, can we have a talk about this we can create a "Global Image System" but in my opinion the images need to be stored at the Region level.
Check the AWS and Azure CSP which are storing images regionally.
If we don't set the image regionally for some CSP could be an issue to address.

Reply:
No issues, I will adopt this part to be regional.